services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/assignmentFlowConstants.tsx

```diff
@@ -37,22 +37,27 @@ export const PRE_AP_WRITINGS_SKILLS_2_UNIT_TEMPLATE_ID = 195
 export const AP_WRITINGS_SKILLS_UNIT_TEMPLATE_ID = 193
 export const SPRING_BOARD_SKILLS_UNIT_TEMPLATE_ID = 253
 
-export const STARTER_DIAGNOSTIC_ACTIVITY_ID = 2537
-export const INTERMEDIATE_DIAGNOSTIC_ACTIVITY_ID = 2539
+export const STARTER_DIAGNOSTIC_ACTIVITY_ID = 2861
+export const INTERMEDIATE_DIAGNOSTIC_ACTIVITY_ID = 2863
 export const ADVANCED_DIAGNOSTIC_ACTIVITY_ID = 2541
 export const ELL_STARTER_DIAGNOSTIC_ACTIVITY_ID = 2550
 export const ELL_INTERMEDIATE_DIAGNOSTIC_ACTIVITY_ID = 2555
 export const ELL_ADVANCED_DIAGNOSTIC_ACTIVITY_ID = 2563
 
+export const OLD_STARTER_DIAGNOSTIC_2024_ACTIVITY_ID = 2537
+export const OLD_INTERMEDIATE_DIAGNOSTIC_2024_ACTIVITY_ID = 2539
+export const OLD_STARTER_DIAGNOSTIC_2024_POST_ACTIVITY_ID = 2538
+export const OLD_INTERMEDIATE_DIAGNOSTIC_2024_POST_ACTIVITY_ID = 2540
+
 export const OLD_STARTER_DIAGNOSTIC_ACTIVITY_ID = 1663
```

Comment:
The plan is to come back through and change this (and the backend equivalent) in the near future, right? Otherwise I think it might just be worth changing to include 2023 in these titles as well.

Reply:
It is the plan to remove this, but I think updating the name here until we do makes sense anyway.  I'll do that.